var/spack/repos/builtin/packages/hipsycl/package.py

```diff
@@ -35,7 +35,9 @@ class Hipsycl(CMakePackage):
     version("0.8.0", commit="2daf8407e49dd32ebd1c266e8e944e390d28b22a", submodules=True)
     version("develop", branch="develop", submodules=True)
 
-    variant("cuda", default=False, description="Enable CUDA backend for SYCL kernels")
+    variant("opencl", default=False, description="Enable OpenCL backend for SYCL kernels")
+    variant("sscp", default=False, description="Enable SSCP compiler")
+    variant("intel", default=False, description="Enable Intel Level Zero backend for SYCL kernels")
```

Comment:
`intel` is confusing. E.g., currently, it is recommended to use SSCP+OpenCL for Intel GPUs, since it is more marure than SSCP+LevelZero. Suggest `level_zero` or `ze` or `l0`.

Reply:
I renamed the `intel` variant `level_zero`.